primitives/core/src/error.rs

```diff
@@ -42,13 +42,18 @@ pub enum ErrorDetail {
 	ParseError,
 	// errors when communicating with data provider, e.g. HTTP error
 	DataProviderError(ErrorString),
-	InvalidIdentity,
+	// error when tee-worker detects that identity verification data is linked to different web2
+	// account than expected
 	WrongWeb2Handle,
```

Comment:
My understanding was that it errors when the provided proof doesn't match the account to be linked. 

The description sounds like "IdentityAlreadyLinked"

Reply:
@jonalvarezz, maybe I should replace word _linked_ and add an example, what do you think about:

	// error when tee-worker detects that identity verification data is related to other web2
	// account than expected, for example wrong tweet id was provided

Reply:
@kziemianek sounds good